.github/workflows/tick-ci.yml

```
@@ -0,0 +1,221 @@
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          default: true
+          components: rustfmt, clippy
+      - name: Install WASM target
+        run: rustup target add wasm32-wasip1
+      - name: Cache cargo dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)